lib/logger/utils.ts

```diff
@@ -32,3 +36,41 @@ export class ErrorStream extends Stream {
     return this._errors;
   }
 }
+
+export function withSanitizer(streamConfig): bunyan.Stream {
+  const stream = streamConfig.stream;
+  if (stream && stream.writable) {
+    const write = (chunk, enc, cb) => {
```

Comment:
I think chunk would always be `BunyanRecord` here. can we simply walk recursive to replace all props of string type?

Reply:
It could be simpler and faster to `JSON.parse(sanitize(JSON.stringify(chunk))` ?

Reply:
Hopefully we don't have cyclic objects